quactography/visu/dist_prob_edge.py

```diff
@@ -114,5 +110,96 @@ def plot_distribution_of_probabilities_edge(
 
     # else:
     #     print(
-    #         "The solution is not in given subset of solutions found by QAOA.\n_________________")
+    #         "The solution is not in given subset of solutions found by QAOA.\n____")
     #     )
+
+ 
+def plot_distribution_comparison(
+    in_file, visu_out_file_selected, save_only
+):
+    """
+    Plot the distribution of probabilities for the optimal path
+    and the selected paths.
+
+    Parameters
+    ----------
+    in_file: str
+        The input file containing the optimization results.
+    visu_out_file_selected: str
+        The output file name for the histogram of selected paths.
+    save_only: bool
+        If True, the figure is saved without displaying it.
+
+    Returns
+    -------
+    None
+    """
+    counts = []
+    sumDir = 0
+    path = Path(in_file)
+
+    glob_path = path.glob('*')
```

Comment:
Isn't `in_file` already a file? Why use glob here? Should `in_file` be `in_folder`?